utils/prometheus/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Jeffery To
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=prometheus
+PKG_VERSION:=2.2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prometheus/prometheus.git
+PKG_SOURCE_VERSION:=f8dcf9b272c9ab1a6148ccc22c4de6d4403c601d
```

Comment:
Would be great if you can include a [`PKG_MIRROR_HASH`](https://openwrt.org/docs/guide-developer/packages#use_source_repository) (and preferable `PKG_SOURCE_DATE` also).

Reply:
Added